services/common/src/redux/actionCreators/permitActionCreator.ts

```diff
@@ -686,3 +687,81 @@ export const updateStandardPermitCondition = (
       .catch(() => dispatch(error(NetworkReducerTypes.UPDATE_PERMIT_CONDITION)))
       .finally(() => dispatch(hideLoading()));
   };
+
+export const fetchPermitConditionTags = (): AppThunk => (dispatch) => {
```

Comment:
Would be a bit of a bigger overhaul, but I think we're tending in the direction lately of using the newer `slice` format for setting up actions like this.  It consolidates things quite a bit, so there's less need to update 4 or 5 different files when you need to make a change.  
This format is still valid, but it's worth considering.